src/templates/macros.java

```diff
@@ -72,6 +73,9 @@
             }
     }
     {% endif %}
+    {%- else %}
+    // Sorry, the callable "{{ callable.name() }}" isn't supported.
+    {%- endif %}
```

Comment:
What does adding this provide?

Reply:
It leaves a comment in the generated code so they know why what they might have expected isn't there.